metis-core/metis-core-common/src/main/java/eu/europeana/metis/core/workflow/plugins/PluginStatus.java

```diff
@@ -7,7 +7,7 @@
  * @since 2017-05-26
  */
 public enum PluginStatus {
-  INQUEUE, RUNNING, FINISHED, CANCELLED, FAILED;
+  INQUEUE, CLEANING, RUNNING, FINISHED, CANCELLED, FAILED;
```

Comment:
There may be some checks (if statements) that need to be changed given the new CLEANING state. I found the following candidates (maybe changing is not strictly necessary in all of them, but it is better to be safe than sorry):
1. WorkflowExecution.setAllRunningAnInqueuePluginsToCancelled
2. WorkflowExecution.checkAndSetAllRunningAndInqueuePluginsToCancelledIfOnePluginHasFailed
3. WorkflowExecutor.runInqueueOrRunningStateWorkflowExecution